sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/AggUtils.scala

```diff
@@ -529,4 +533,13 @@ object AggUtils {
       case None => partialAggregate
     }
   }
+
+  /**
+   * Returns whether a sort aggregate should be force applied.
+   * The config key is hard-coded because it's testing only and should not be exposed.
+   */
+  private def forceApplySortAggregate(conf: SQLConf): Boolean = {
+    Utils.isTesting &&
+      conf.getConfString("spark.sql.aggregate.forceApplySortAggregate", "false") == "true"
```

Comment:
I'd use `spark.sql.test.forceApplySortAggregate` to make the intention clear

Reply:
@cloud-fan - I was actually following same naming as [`spark.sql.join.forceApplyShuffledHashJoin`](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/joins.scala#L446), but I agree `.test.` is better. Will update.

Reply:
@cloud-fan - updated.